src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -877,20 +877,20 @@ reference:
       ;
 parameters: LEFT_BRACE (ARGUMENT | COMMA | WS | NEWLINE | LEADING_ASTERISK)* RIGHT_BRACE;
 
-javadocTag: AUTHOR_LITERAL (WS | NEWLINE)* ((WS | NEWLINE) description)?
+javadocTag: AUTHOR_LITERAL (WS | NEWLINE)+ description
 
     | DEPRECATED_LITERAL (WS | NEWLINE)* ((WS | NEWLINE) description)?
 
-      | EXCEPTION_LITERAL (WS | NEWLINE)* ((WS | NEWLINE) CLASS_NAME)? (WS | NEWLINE)*
-          ((WS | NEWLINE) description)?
+      | EXCEPTION_LITERAL (WS | NEWLINE | {!isNextJavadocTag()}? LEADING_ASTERISK)+ CLASS_NAME
```

Comment:
Why add LEADING_ASTERISK here? Is there a corresponding tets case?
If LEADING_ASTERISK is needed I think it should be a separate issue.

Reply:
Yes it is needed I will open an issue and add a test case, none verify this currently.
https://4942-single-jcheck.netlify.com/apache-ant/#A18

Reply:
#5035